### PR TITLE
Allow esmtp access .esmtprc BZ(1691149)

### PR DESCRIPTION
**Pull request by zpytela**
Signed-off-by: Zdenek Pytela <zpytela@redhat.com>

### CODE REVIEW

**Comment by zpytela** [comment]
Fedoras 28-30 are affected.

Scratchbuild finished, selinux-policy-3.14.4-3.1691149.200.fc31 created.
rpm -qa "selinux-policy*"; matchpathcon /home/a/.esmtprc /root/.esmtprc; sesearch -T -s system_mail_t -t user_home_dir_t -c file; sesearch -T -s system_mail_t -t admin_home_t -c file

Before:
selinux-policy-3.14.4-5.fc31.noarch
selinux-policy-targeted-3.14.4-5.fc31.noarch
selinux-policy-devel-3.14.4-5.fc31.noarch
selinux-policy-doc-3.14.4-5.fc31.noarch
/home/a/.esmtprc        unconfined_u:object_r:user_home_t:s0
/root/.esmtprc  system_u:object_r:admin_home_t:s0
type_transition system_mail_t user_home_dir_t:file mail_home_rw_t ".esmtp_queue";
type_transition system_mail_t user_home_dir_t:file mail_home_t ".forward";
type_transition system_mail_t user_home_dir_t:file mail_home_t ".mailrc";
type_transition system_mail_t user_home_dir_t:file mail_home_t "dead.letter";
type_transition system_mail_t admin_home_t:file mail_home_rw_t ".esmtp_queue";
type_transition system_mail_t admin_home_t:file mail_home_t ".forward";
type_transition system_mail_t admin_home_t:file mail_home_t ".mailrc";
type_transition system_mail_t admin_home_t:file mail_home_t "dead.letter";
type_transition system_mail_t admin_home_t:file mail_home_t;

After:
selinux-policy-devel-3.14.4-5.1691149.200.fc31.noarch
selinux-policy-3.14.4-5.1691149.200.fc31.noarch
selinux-policy-doc-3.14.4-5.1691149.200.fc31.noarch
selinux-policy-targeted-3.14.4-5.1691149.200.fc31.noarch
/home/a/.esmtprc        unconfined_u:object_r:mail_home_t:s0
/root/.esmtprc  system_u:object_r:mail_home_t:s0
type_transition system_mail_t user_home_dir_t:file mail_home_rw_t ".esmtp_queue";
type_transition system_mail_t user_home_dir_t:file mail_home_t ".esmtprc";
type_transition system_mail_t user_home_dir_t:file mail_home_t ".forward";
type_transition system_mail_t user_home_dir_t:file mail_home_t ".mailrc";
type_transition system_mail_t user_home_dir_t:file mail_home_t "dead.letter";
type_transition system_mail_t admin_home_t:file mail_home_rw_t ".esmtp_queue";
type_transition system_mail_t admin_home_t:file mail_home_t ".esmtprc";
type_transition system_mail_t admin_home_t:file mail_home_t ".forward";
type_transition system_mail_t admin_home_t:file mail_home_t ".mailrc";
type_transition system_mail_t admin_home_t:file mail_home_t "dead.letter";
type_transition system_mail_t admin_home_t:file mail_home_t;

Please review the PR.

File: mta.fc [hunk 1]
```diff
@@ -18,8 +19,9 @@ ifdef(`distro_redhat',`
 
 /root/\.forward		--	gen_context(system_u:object_r:mail_home_t,s0)
 /root/dead\.letter	--	gen_context(system_u:object_r:mail_home_t,s0)
+/root/\.esmtprc		--	gen_context(system_u:object_r:mail_home_t,s0)
 /root/\.mailrc		--	gen_context(system_u:object_r:mail_home_t,s0)
-/root/\.esmtp_queue(/.*)?     gen_context(system_u:object_r:mail_home_rw_t,s0)
+/root/\.esmtp_queue(/.*)?	gen_context(system_u:object_r:mail_home_rw_t,s0)
```
Comment (wrabcak) [inline mta.fc]: Why is this file modified?

**Comment by zpytela** [comment]
Formatting (spaces to tab).
